pkg/controller/registry/resolver/cache/operators.go

```diff
@@ -140,13 +140,20 @@ type OperatorSourceInfo struct {
 	StartingCSV    string
 	Catalog        SourceKey
 	DefaultChannel bool
+	Deprecations   Deprecations
```

Comment:
Also make this a pointer?
```suggestion
	Deprecations   *Deprecations
```